### DISCUSSION TITLE
What is the unit of ``network_rx'' and ``network_tx''

### DISCUSSION THREAD

**Discussion by RayShing** [nicolargo/glances#2093] [General]
**Is your feature request related to a problem? Please describe.**
I store the data into the csv file. Now I want to know the throughput of the network tranmisstion. I think I should focus on the "network_rx" and "network_tx" field. But I am confused the unit of these two fields. It is bit/s or byte/s or something else?

![WX20220718-164816](https://user-images.githubusercontent.com/61814345/179614726-5e334f12-8940-4578-8296-7171d96e8c48.png)

**Comment by nicolargo** [comment]
It is bit per second (bps) (source:https://github.com/nicolargo/glances/blob/develop/glances/plugins/glances_network.py#L34-L35)
